packages/terra-notification-dialog/src/NotificationDialog.module.scss

```diff
@@ -2,25 +2,25 @@
 
 :local {
   .notification-dialog {
-    background: var(--notification-dialog-background, #fff);
-    border-radius: var(--notification-dialog-border-radius, 5px);
+    background: var(--terra-notification-dialog-background, #fff);
+    border-radius: var(--terra-notification-dialog-border-radius, 5px);
     display: flex;
     flex-direction: column;
     position: fixed;
 
     @media screen and (max-width: 768px) {
-      width: calc(100% - 2 * var(--notification-dialog-small-horizontal-inset, 10px));
+      width: calc(100% - 2 * var(--terra-notification-dialog-small-horizontal-inset, 10px));
```

Comment:
Should this be `--terra-notification-dialog-small-horizontal-width` or `--terra-notification-dialog-small-horizontal-inset-width`?

Reply:
I think adding `width` might cause an issue of misinformation with theming since its not actually setting width its setting a variable that width is calculated off of. Though that might be mitigated by the variable only being used here and nowhere else.

Reply:
Ok, yeah that makes sense. I'm fine leaving this as-is.